.azure/pipelines/ci.yml

```diff
@@ -733,7 +733,7 @@ stages:
     parameters:
       platform:
         name: 'Managed'
-        container: 'mcr.microsoft.com/dotnet-buildtools/prereqs:centos-7-20210714125435-9b5bbc2'
+        container: 'mcr.microsoft.com/dotnet-buildtools/prereqs:fedora-33-20210222183538-031e7d2'
```

Comment:
Any chance we could use an image that's still in support e.g. Fedora 35❔

Reply:
Per source-build sync, this morning, we are updating this to centos-stream8: https://github.com/dotnet/installer/blob/0600c205ab64abf7caecc6dee9d89514aeddb458/src/SourceBuild/Arcade/eng/common/templates/job/source-build-run-tarball-build.yml#L15

Just updated the PR